modules/document_repository/php/document_repository.class.inc

```diff
@@ -78,10 +78,9 @@ class Document_Repository extends \NDB_Menu_Filter
                           'v.Data_dir',
                           'v.Date_uploaded',
                          );
-
-        $openAccordion = $_GET['openAccordion'];
+        $openAccordion = isset($_GET['openAccordion']) ?? false;
         $this->tpl_data['openAccordion'] = $openAccordion;
-        $filtered = $_GET['filtered'];
+        $filtered = isset($_GET['filtered']) ?? null;
```

Comment:
i think this should be `$filtered = $_GET['filtered'] ?? null`

Reply:
Any update on this? (And isn't that also true of the line above?)